sklearn/inspection/_permutation_importance.py

```diff
@@ -98,6 +98,32 @@ def permutation_importance(estimator, X, y, scoring=None, n_repeats=5,
     ----------
     .. [BRE] L. Breiman, "Random Forests", Machine Learning, 45(1), 5-32,
              2001. https://doi.org/10.1023/A:1010933404324
+
+    Examples
+    --------
+    >>> from sklearn.linear_model import LogisticRegression
+    >>> from sklearn.inspection import permutation_importance
+
```

Comment:
If you can remove all blank lines between each `>>>`. They will create some blocks on the html page (cf. https://93833-843222-gh.circle-artifacts.com/0/doc/modules/generated/sklearn.inspection.permutation_importance.html)